### ISSUE TITLE
tpcc check can't check all warehouse

### ISSUE THREAD

**Issue by ChenPeng2013** [pingcap/go-tpc#68]
### 1. Minimal reproduce step (Required)
tidb cluster v4.0.9
```
go-tpc tpcc prepare --warehouses 28000
go-tpc tpcc check --warehouses 28000 --threads 10
```

### 2. What did you expect to see? (Required)
```
begin to check warehouse 1 at ...
...
begin to check warehouse 28000 at ...
Finished
```

### 3. What did you see instead (Required)
```
./go-tpc tpcc check --warehouses 28000 --threads 10
begin to check warehouse 7 at condition 3.3.2.1
begin to check warehouse 3 at condition 3.3.2.8
begin to check warehouse 1 at condition 3.3.2.10
begin to check warehouse 8 at condition 3.3.2.8
begin to check warehouse 6 at condition 3.3.2.8
begin to check warehouse 2 at condition 3.3.2.5
begin to check warehouse 5 at condition 3.3.2.12
begin to check warehouse 10 at condition 3.3.2.9
begin to check warehouse 4 at condition 3.3.2.6
begin to check warehouse 9 at condition 3.3.2.7
begin to check warehouse 7 at condition 3.3.2.8
begin to check warehouse 10 at condition 3.3.2.10
begin to check warehouse 6 at condition 3.3.2.9
execute check failed, err check warehouse 2 at condition 3.3.2.5 failed count(*) in warehouse 2, but got 9000.000000
begin to check warehouse 8 at condition 3.3.2.10
begin to check warehouse 3 at condition 3.3.2.9
begin to check warehouse 3 at condition 3.3.2.1
begin to check warehouse 3 at condition 3.3.2.3
begin to check warehouse 3 at condition 3.3.2.4
begin to check warehouse 4 at condition 3.3.2.8
begin to check warehouse 7 at condition 3.3.2.9
begin to check warehouse 6 at condition 3.3.2.12
begin to check warehouse 4 at condition 3.3.2.3
begin to check warehouse 4 at condition 3.3.2.2
begin to check warehouse 4 at condition 3.3.2.4
execute check failed, err check warehouse 5 at condition 3.3.2.12 failed count(*) in warehouse 5, but got 9000.000000
begin to check warehouse 9 at condition 3.3.2.9
execute check failed, err check warehouse 10 at condition 3.3.2.10 failed count(*) in warehouse 10, but got 9000.000000
execute check failed, err check warehouse 1 at condition 3.3.2.10 failed count(*) in warehouse 1, but got 9000.000000
execute check failed, err check warehouse 8 at condition 3.3.2.10 failed count(*) in warehouse 8, but got 9000.000000
begin to check warehouse 4 at condition 3.3.2.5
begin to check warehouse 7 at condition 3.3.2.12
begin to check warehouse 3 at condition 3.3.2.5
begin to check warehouse 9 at condition 3.3.2.12
execute check failed, err check warehouse 4 at condition 3.3.2.5 failed count(*) in warehouse 4, but got 9000.000000
execute check failed, err check warehouse 3 at condition 3.3.2.5 failed count(*) in warehouse 3, but got 9000.000000
execute check failed, err check warehouse 6 at condition 3.3.2.12 failed count(*) in warehouse 6, but got 9000.000000
execute check failed, err check warehouse 7 at condition 3.3.2.12 failed count(*) in warehouse 7, but got 9000.000000
execute check failed, err check warehouse 9 at condition 3.3.2.12 failed count(*) in warehouse 9, but got 9000.000000
Finished
```

**Comment by mahjonp** [comment]
I will take a look soon

**Comment by mahjonp** [comment]
I think this issue should have been resolved by 08b834760234cb37ea4ce90193eec29547b240fd
